serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1889,17 +1890,26 @@ export class _Exporter {
             let inverseBindMatrices: Matrix[] = [];
             let skeletonMesh = babylonScene.meshes.find((mesh) => { mesh.skeleton === skeleton; });
             skin.skeleton = skeleton.overrideMesh === null ? (skeletonMesh ? nodeMap[skeletonMesh.uniqueId] : undefined) : nodeMap[skeleton.overrideMesh.uniqueId];
+            let boneIndexMap: Map<number, Bone> = new Map();

```

Comment:
Careful with using Map as they are not fully supported in legacy browsers. Might consider using a normal object for this.

Reply:
thanks, replaced with a generic object instead